app/assets/javascripts/discourse/app/lib/topic-list-tracker.js

```diff
@@ -1,8 +1,25 @@
 import { Promise } from "rsvp";
 let model, currentTopicId;
 
+let highestReadCache = {};
```

Comment:
Any time we store data in a module we need a corresponding `reset` method which is called after tests. Otherwise we leak state and that causes flaky tests.

Reply:
Thank you for that tip, it is fixed :)